.yamllint.yaml

```diff
@@ -1,4 +1,13 @@
 extends: default
 
 rules:
-  line-length: disable
\ No newline at end of file
+  line-length: disable
+  comments:
```

Comment:
There was an error message running ansible-lint listing them. ansible-lint also internally run yamllint, but Ansible likes its rules. The ansible-lint warning was suggesting to remove this file or tune it to match the Ansible rules